src/wallet/rpc/coins.cpp

```diff
@@ -21,7 +21,7 @@ static CAmount GetReceived(const CWallet& wallet, const UniValue& params, bool b
     std::vector<CTxDestination> addresses;
     if (by_label) {
         // Get the set of addresses assigned to label
-        addresses = wallet.ListAddrBookAddresses(CWallet::AddrBookFilter{LabelFromValue(params[0])});
+        addresses = wallet.ListAddrBookAddresses(CWallet::AddrBookFilter{CHECK_NONFATAL(LabelFromValue(params[0]))});
```

Comment:
Since `AddrBookFilter` takes a `std::optional<std::string> m_op_label`, I think there's no need for the `CHECK_NONFATAL` here? Wouldn't be any behaviour change either I think, because the existing code would throw if params[0] were to be null? (not 100% sure on that)

```suggestion
        addresses = wallet.ListAddrBookAddresses(CWallet::AddrBookFilter{LabelFromValue(params[0])});
```